### PR TITLE
LPS-147977 Reorganize Page Editor CSS

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_variables.scss [hunk 1]
```diff
@@ -8,19 +8,10 @@ $pageEditorSpacing: 8px;
...
 
-// z-indexes, ordered numerically.
-
-$dragSourceZIndex: 977;
-$translationStatusListZIndex: 977;
 $portletTopperZIndex: 986;
-$sidebarZIndex: 988;
-$dragPreviewLayer: 991;
-$undoHistoryOverlayZIndex: 1001;
-$modalZIndex: 1050;
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/_variables.scss:26]: Now that you say it, I agree. Changed 🥰

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
Screenshot 2022-02-25 at 08 27 21
@sandrodw3  I think we cut too much CSS from the PreviewModal file. 

From my tests only these lines needs to be added:

```css
.modal-body {
	background-color: $cadmin-gray-400;
	display: flex;
	justify-content: center;
}
```

And now I leave because I just realized @veroglez assigned this pr :p

**Comment by sandrodw3** [comment]
> Screenshot 2022-02-25 at 08 27 21 @sandrodw3 I think we cut too much CSS from the PreviewModal file.
> 
> From my tests only these lines needs to be added:
> 
> ```css
> .modal-body {
> 	background-color: $cadmin-gray-400;
> 	display: flex;
> 	justify-content: center;
> }
> ```
> 
> And now I leave because I just realized @veroglez assigned this pr :p

How did you reproduce this? 😩

**Comment by veroglez** [comment]
LGTM! 👏

**Comment by veroglez** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:reevaluate:1297401_1854

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**